frontend/packages/console-app/src/components/volume-modes/volume-mode.tsx

```diff
@@ -20,17 +20,24 @@ export const VolumeModeSelector: React.FC<VolumeModeSelectorProps> = (props) =>
     storageClass,
     availableVolumeMode,
     loaded,
+    spAccessMode,
+    spVolumeMode,
```

Comment:
```suggestion
    volumeMode,
```
can we just add initial value (e.g. volumeMode) and use ecisting access mode prop ?

Reply:
also, volumeMode is already had it as the state
and I'll try to use the existing prop for accessMode

Reply:
changed to `volMode`
also using existing prop